platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/LoggerImpl.kt

```diff
@@ -265,6 +266,18 @@ internal class LoggerImpl(
         appExitSaveCurrentSessionId()
     }
 
+    override fun shutdown() {
+        runCatching {
+            appExitLogger.uninstallAppExitLogger()
+            eventsListenerTarget.stop()
+            // TODO(murki): stop the rest of the threads (replay, resource utilization, etc.)
+            processingQueue.shutdown()
+            // TODO(murki): Should we run this? in a bg thread?
+            CaptureJniLibrary.flush(this.loggerId, true)
```

Comment:
Running it blocking will potentially mean that it can take a while, maybe we just start with running it async and see how that goes?

Reply:
Another option is to move this into the Rust layer so that it can be sequenced as part of shutting down, though it would mean having to rework the shutdown logic slightly

Reply:
yeah I think for now I'll remove explicitly calling it and later we can rework it into the shutdown sequence if needed